shared/uavobjectdefinition/hwsprf3e.xml

```diff
@@ -0,0 +1,142 @@
+<?xml version="1.0"?>
+<xml>
+	<object name="HwSprf3e" singleinstance="true" settings="true" category="HardwareSettings">
+		<description>Selection of optional hardware configurations.</description>
+		<field name="Uart1" units="function" type="enum" elements="1" parent="HwShared.PortTypes" defaultvalue="Disabled">
+			<description/>
```

Comment:
Isn't it possible to describe where the port is located ("Left of/Right of/Next to") ?